### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by amirmc**
Edited initial paragraph. Commented out the other material and
instead pointed at the main website.

### CODE REVIEW

**Comment by amirmc** [comment]
I left a bunch of stuff commented out. Happy to remove it but wanted to check first.

File: README.md [hunk 1]
```diff
@@ -21,16 +21,16 @@ available online.
...
 
-There are several diverse backends in Mirage that require rather specialised
+There are several diverse backends in MirageOS that require rather specialised
 build steps (from JavaScript to Xen unikernels), and this complexity is wrapped
 up in the tool.
 
 To work with Mirage, you'll need the following prerequisites installed:
 
-* a working [OCaml](http://ocaml.org) compiler (4.01.0 or higher).
+* a working [OCaml](http://ocaml.org) compiler (4.02.1 or higher).
```
Comment (avsm) [inline README.md]: 4.02.3

**Comment by avsm** [comment]
a short thing about the `docker run -it unikernel/mirage` container would also be useful. I'll be maintaining that moving forward.

File: README.md (hunk 1 above)
Comment (hannesm) [inline README.md]: 4.02.2 should be fine!? (that's the lower bound of ppx sexp)...

**Comment by hannesm** [comment]
shouldn't docker instructions be kept in some (to-be-written) wiki/install-docker?  there shouldn't be the impression that MirageOS depends on docker...

**Comment by amirmc** [comment]
I think it's fine to mention the Docker images since the aim is to maintain them.  If it makes it quicker to get started (for those who already have Docker) then it's worth highlighting them.

**Comment by avsm** [comment]
> shouldn't docker instructions be kept in some (to-be-written) wiki/install-docker? 

just as a "quickstart" in the README is fine, along with manual install instructions.  The OPAM PPAs are getting increasingly difficult to maintain and these are the ones used in bulk builds, so most likely to work...

**Comment by avsm** [comment]
re: 4.02.2 vs 4.02.3, again only the latest point release is tested, so best to recommend that as the lower bound

**Comment by avsm** [comment]
also need to do a quick sweep for other compiler version deps -- there is a version check on opam in `mirage` but not sure about compiler version. @samoht may remember

**Comment by samoht** [comment]
I've just added more checks to the tool (mirage-types, mirage-types-lwt, and OCaml version) so should be better. Merging now.

**Pull request merged**